data/css/_colors.scss

```diff
@@ -1,3 +1,5 @@
+$brand-color: #005a71 !default;
```

Comment:
Wait, is your concern just that it is being added to the `_colors.scss` file (and thus the default theme) as opposed to just living in `course.scss`? Because that I can change by just swapping variable names. As I see it right now, these variables have no consistent meaning: `$primary-medium-color` doesn't have the same semantic meaning across apps - it's just one of the 7 color variables you can use for styling an app.